hadoop-hdds/docs/content/beyond/Containers.zh.md

```diff
@@ -0,0 +1,212 @@
+
+```
+OZONE-SITE.XML_ozone.om.address=om
+OZONE-SITE.XML_ozone.om.http-address=om:9874
+OZONE-SITE.XML_ozone.scm.names=scm
+#...
+```
+
+你可以看到我们所使用的命名规范，根据这些环境变量的名字，`hadoop-runner` 基础镜像中的[脚本](https://github.com/apache/hadoop/tree/docker-hadoop-runner-latest/scripts) 会生成合适的 hadoop XML 配置文件（在我们这种情况下就是 `ozone-site.xml`）。
+
+`hadoop-runner` 镜像的[入口点](https://github.com/apache/hadoop/blob/docker-hadoop-runner-latest/scripts/starter
+.sh)包含了一个辅助脚本，这个辅助脚本可以根据环境变量触发上述的配置文件生成以及其它动作（比如初始化 SCM 和 OM 的存储、下载必要的 keytab 等）。
+
+## 测试 
+
+`docker-compose` 的方式应当只用于本地测试，不适用于多节点集群。要在多节点集群上使用容器，我们需要像 Kubernetes 这样的容器编排系统。
+
+Kubernetes 示例文件在 `kubernetes` 文件夹中。
+
+*请注意*：所有提供的镜像都使用 `hadoop-runner` 作为基础镜像，这个镜像中包含了所有测试环境所需的测试工具。对于生产环境，我们推荐用户使用自己的基础镜像创建可靠的镜像。
+
+### 发行包测试
+
+可以通过部署任意的示例集群来测试发行包：
+
+```bash
+cd kubernetes/examples/ozone
+kubectl apply -f
+```
+
+注意，此时会从 Docker Hub 下载最新的镜像。
+
+### 开发构建测试
+
+为了测试开发中的构建，你需要创建自己的镜像并上传到自己的 docker 仓库中：
+
+
+```bash
+mvn clean install -DskipTests -Pdocker-build,docker-push -Ddocker.image=myregistry:9000/name/ozone
+```
+
+所有生成的 kubernetes 资源文件都会使用这个镜像 (`image:` keys are adjusted during the build)
+
+```bash
+cd kubernetes/examples/ozone
+kubectl apply -f
+```
+
+## 生产
+
+<div class="alert alert-danger" role="alert">
+我们<b>强烈</b>推荐在生产集群使用自己的镜像，并根据实际的需求调整基础镜像、文件掩码、安全设置和用户设置。
+</div>
+
+你可以使用我们开发中所用的镜像作为示例：
+
+ * [基础镜像] (https://github.com/apache/hadoop/blob/docker-hadoop-runner-jdk11/Dockerfile)
+ * [完整镜像] (https://github.com/apache/hadoop/blob/trunk/hadoop-ozone/dist/src/main/docker/Dockerfile)
+
+ Dockerfile 中大部分内容都是可选的辅助功能，但如果要使用我们提供的 kubernetes 示例资源文件，你可能需要[这里](https://github.com/apache/hadoop/tree/docker-hadoop-runner-jdk11/scripts)的脚本。
+
+  * 两个 python 脚本将环境变量转化为实际的 hadoop XML 配置文件
+  * start.sh 根据环境变量执行 python 脚本（以及其它初始化工作）
+
+## 容器
+
+Ozone 相关的容器镜像和 Dockerfile 位置：
+
+
+<table class="table table-dark">
+  <thead>
+    <tr>
+      <th scope="col">#</th>
+      <th scope="col">容器</th>
+      <th scope="col">仓库</th>
+      <th scope="col">基础镜像</th>
+      <th scope="col">分支</th>
+      <th scope="col">标签</th>
+      <th scope="col">说明</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th scope="row">1</th>
+      <td>apache/ozone</td>
+      <td>https://github.com/apache/hadoop-docker-ozone</td>
+      <td>ozone-... </td>
+      <td>hadoop-runner</td>
+      <td>0.3.0,0.4.0,0.4.1</td>
+      <td>每个 Ozone 发行版都对应一个新标签。</td>
+    </tr>
+    <tr>
+      <th scope="row">2</th>
+      <td>apache/hadoop-runner </td>
+      <td>https://github.com/apache/hadoop</td>
+      <td>docker-hadoop-runner</td>
+      <td>centos</td>
+      <td>jdk11,jdk8,latest</td>
+      <td>这是用于测试 Hadoop Ozone 的基础镜像，包含了一系列可以让我们更加方便地运行 Ozone 的工具。
+      </td>
+    </tr>
+    <!---tr>
+      <th scope="row">3</th>
+      <td>apache/ozone:build (WIP)</td>
+      <td>https://github.com/apache/hadoop-docker-ozone</td>
+      <td>ozone-build </td>
+      <td> </td>
+      <td> </td>
+      <td>TODO: 此处需要添加更多文档</td>
+    </tr-->
```

Comment:
Could we remove this part ?

Reply:
@xiaoyuyao @elek could we just remove this line or add something else ?

Reply:
Sure, feel free to remove it. I will create a patch as the previous lines are also outdated...